.github/workflows/generate.yml

```diff
@@ -30,6 +30,7 @@ jobs:
       - name: Generate all assets
         env:
           ZERION_API_KEY: ${{ secrets.ZERION_API_KEY }}
+          COINCAP_API_KEY: ${{ secrets.COINCAP_API_KEY }}
```

Comment:
So does it mean we need this key locally if we run `yarn generate:all` or there is a risk of chance it fails?

Reply:
Also, should we reflect the difference between VITE_COINCAP_API_KEY and COINCAP_API_KEY somewhere? (maybe in the naming...)